tests/plugins/pastefromwordimage/manual/pasteimagewithlink.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: bug, 4.8.0, 662, pastefromwordimage
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, pastefromwordimage, sourcearea, elementspath, newpage, resize, link
+
+----
+
+1. Open document and copy whole content.
+1. Clear editor content.
+1. Paste copied content into CKEditor.
+
+### Perform above steps for all files:
+  1. [Link_image.docx](../generated/_fixtures/Link_image/Link_image.docx)
+
+**Expected:** Images is visible and pasted only once. Image is wrapped with an anchor link.
```

Comment:
It appear that `image` plugin was missing in those test. Because PFW Image register img tag as allowed, the img appear but had stripped away width and height.